<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>371</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>551</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="installdir">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Install Directory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="installbtn">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Install</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browse_dir">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>215</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="pic_logo">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="titlebar_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="titlebar_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
